Marlin/configuration_store.cpp

```diff
@@ -36,10 +36,10 @@
  *
  */
 
-#define EEPROM_VERSION "V23"
+#define EEPROM_VERSION "V24"
```

Comment:
I like the possibility to tweak during runtime the value (this was a great idea), but who may tweak this ? Developers and Host software which IMO is highly "session" dependent thus not really requiring to be remembered between power cycles.